src/textual/widgets/_tabbed_content.py

```diff
@@ -116,21 +113,16 @@ def __init__(self, *titles: TextType, initial: str = "") -> None:
         self._initial = initial
         super().__init__()
 
-    def validate_active(self, active: str) -> str:
-        """It doesn't make sense for `active` to be an empty string.
-
-        Args:
-            active: Attribute to be validated.
-
-        Returns:
-            Value of `active`.
+    @property
+    def active(self) -> str:
+        """The ID of the active tab, or empty string if none are active."""
+        return self.get_child_by_type(Tabs).active
 
-        Raises:
-            ValueError: If the active attribute is set to empty string.
-        """
+    @active.setter
+    def active(self, active: str) -> None:
         if not active:
             raise ValueError("'active' tab must not be empty string.")
```

Comment:
Originally I just let the setter set the attribute `active` on the child `Tabs` but the tests failed because they expected a `ValueError` when setting the tab to an empty string, so I added it back in 6eb141c.

However, to me it doesn't make much sense that I can't set `TabbedContent.active = ""` but _**I can**_ set `Tabs.active = ""`.
Assuming you agree, should I change the validation in `Tabs` so that it checks for an empty active tab ID at that point?

Reply:
@willmcgugan pinging you because you already merged but this was left unresolved.